Testing/functional/tests/cypress/integration/ui/services/view.js

```diff
@@ -19,10 +19,6 @@ describe("Authenticated Services View", () => {
         cy.visit(servicesTestsConstants.servicesUrl);
```

Comment:
In cy.login you can use /services instead of /home?  Is that not possible?

<img width="1318" alt="Screenshot 2023-03-10 at 2 05 34 PM" src="https://user-images.githubusercontent.com/58790456/224437308-d11947f0-6f54-495a-8b9e-92b32fb3fcc1.png">

Reply:
I'll add one it currently goes to the unauthorized page, which is the same behaviour as dependents.